src/niswitch/examples/niswitch_connect_channels_grpc.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
+
+import argparse
+import grpc
+import niswitch
+import sys
+
+
+def example(resource_name, channel1, channel2, topology, simulate):
+    # if we are simulating resource name must be blank
+    resource_name = '' if simulate else resource_name
+    # Connect to an already running server using the following server information
+    server_address = 'localhost'
+    server_port = '31763'
+    session_name = ''  # user-specified name; empty string means use a new, unnamed session
+    channel = grpc.insecure_channel(f'{server_address}:{server_port}')
+    session_options = niswitch.GrpcSessionOptions(channel, session_name, niswitch.SessionInitializationBehavior.AUTO)
+
+    with niswitch.Session(resource_name=resource_name, topology=topology, simulate=simulate, _grpc_options=session_options) as session:
+        session.connect(channel1=channel1, channel2=channel2)
+        print('Channel ', channel1, ' and ', channel2, ' are now connected.')
+        session.disconnect(channel1=channel1, channel2=channel2)
+        print('Channel ', channel1, ' and ', channel2, ' are now disconnected.')
```

Comment:
Do any of NI-Switch's shipping examples connect and then immediately disconnect like this? If you haven't checked out the LV and C shipping examples, it would be worthwhile to see what use cases they demonstrate.

Reply:
He based it on an existing nimi-python switch example that's in the same directory.

Reply:
Ok, nevermind.